src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -158,7 +161,7 @@ def load_arguments(self, _):
         c.argument('fields', arg_type=feature_fields_arg_type)
 
     with self.argument_context('appconfig feature set') as c:
-        c.argument('feature', help='Name of the feature flag to be set.')
+        c.argument('feature', help="Name of the feature flag to be set. Only alphanumeric characters, '.', '-' and '_' are allowed.")
         c.argument('label', help="If no label specified, set the feature flag with null label by default")
```

Comment:
Should we add a validator for the feature name?

Reply:
As discussed offline, will have a separate PR to implement validation of keys and feature names.

---
In reply to: [355687066](https://github.com/Azure/azure-cli/pull/11467#discussion_r355687066) [](ancestors = 355687066)